deployments/fargate/README.md

```diff
@@ -0,0 +1,95 @@
+# AWS Fargate Deployment
+Familiarity with AWS Fargate (Fargate) is assumed. Consult the 
+[User Guide for AWS Fargate](https://docs.aws.amazon.com/AmazonECS/latest/userguide/what-is-fargate.html) for further reading.
+
+The
+[Splunk OpenTelemetry Collector](https://github.com/signalfx/splunk-otel-collector)
+(splunk-collector or collector) can be deployed to an ECS Task as a standalone container 
+or as sidecar container alongside the application containers that you want to monitor.
+
+Find the Docker image repository for the collector
+[here](https://quay.io/repository/signalfx/splunk-otel-collector?tab=tags).
+The latest tag is `quay.io/signalfx/splunk-otel-collector:latest`.
+
+## Standalone Container
```

Comment:
Suggested change: "To deploy AWS Fargate as a standalone container, add the container definition for the collector to a task that is separate from the task containing the application containers that you 
want to monitor. "

Reply:
I rewrote the entire readme